src/solvation/cosmo.F90

```diff
@@ -1850,7 +1850,14 @@ subroutine cosini(l_print)
       return
     end if
     if (.not. mozyme) then
-      if (lenabc > 22000) then
+      allocate(abcmat(lenabc), xsp(3, lenabc), nset(nppa*numat), bh(lenabc), stat = j)
+      allocate(bmat(lm61, lenabc), stat = i)
+      j = j + i
+      allocate(amat((lenabc*(lenabc + 1))/2), stat = i)
+      j = j + i
+      allocate(cmat((lm61*(lm61 + 1))/2), stat = i)
+      j = j + i
+      if (j /= 0) then
```

Comment:
Why are you moving the `allocate` statements? The functional change is not obvious

Reply:
I didn't move any `allocate` statements. I moved an error message, from being triggered by an artificial size limit (`if (lenabc > 22000)`) to being triggered by a memory allocation error (`if (j /= 0)`). The git diff functionality shows this as the `allocate` statements moving, but what I moved was the block printing the error message. I assume that the +/- highlighting feature identifies the "moved" portion to be whatever is the smallest block of code, but there is an arbitrariness to this highlighting.